lib/block_view_types.go

```diff
@@ -977,6 +977,12 @@ type UtxoOperation struct {
 	// CoinsInCirculation and NumberOfHolders prior to a lockup transaction.
 	PrevLockedBalanceEntry *LockedBalanceEntry
 
+	// PrevLockupYieldCurvePoint and PrevLockupTransferRestriction are
+	// the previous yield curve and transfer restrictions associated
+	// with an UpdateCoinLockupParams transaction.
+	PrevLockupYieldCurvePoint     *LockupYieldCurvePoint
+	PrevLockupTransferRestriction TransferRestrictionStatus
```

Comment:
I moved these around in the struct definition so it aligned with the order in which we encode & decode these attributes.